src/main/java/de/uos/informatik/ko/rcp/geneticalgorithm/GeneticAlgorithm.java

```diff
@@ -31,6 +32,12 @@
         // Only for debug purposes
         var updateDeltas = new LinkedHashMap<Long, Integer>(); // <update delta, new makespan>
 
+        boolean fixedTournamentSize = false;
+        String tournament = Config.instance().parentSelection.name();
+        if(tournament.equalsIgnoreCase(Config.ParentSelection.FIXED_SIZE.name())){
+            fixedTournamentSize = true;
+        }
+
         // bestimme Wkeit (Wert zwischen 0 und 1) dass eine Mutation auftritt
         final double mutationswkeit = 0.4;
```

Comment:
I think it would make sense to use Config.mutationProbability here.

Reply:
We wanted the branches to have one purpose only and did not change several aspects at one time. By merging the branches into the master, we will achieve the desired functionality